packages/jss-plugin-syntax-nested/src/index.test.js

```diff
@@ -637,4 +637,32 @@ describe('jss-plugin-syntax-nested', () => {
       `)
     })
   })
+
+  describe('nest rules inside media query', () => {
+    let sheet
+
+    beforeEach(() => {
+      const localJss = create(settings).use(jssExtend(), nested())
```

Comment:
I don't think we need jss-extend in this test. Please remove

Reply:
cool, it makes sense.